Packet++/src/SSLHandshake.cpp

```diff
@@ -1346,6 +1346,8 @@ SSLServerHelloMessage::SSLServerHelloMessage(uint8_t* data, size_t dataLen, SSLH
 		}
 
 		m_ExtensionList.pushBack(newExt);
+		if (newExt->getTotalLength() <= 0)
```

Comment:
I already fixed that in `master`: https://github.com/seladb/PcapPlusPlus/blob/f30de1387acc1f823e0d4779e5565f4cc32dd4a6/Packet%2B%2B/src/SSLHandshake.cpp#L1258

Reply:
I just realized I only fixed it for the client-hello message and this fix is for server-hello